src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -728,6 +728,7 @@ private BoundExpression BindTupleExpression(TupleExpressionSyntax node, Diagnost
             if (hasNaturalType)
             {
                 tupleTypeOpt = TupleTypeSymbol.Create(node.Location, elements, locations, elementNamesArray, this.Compilation, node, diagnostics);
+                Binder.ReportDiagnosticsIfObsoleteInternal(diagnostics, tupleTypeOpt, node, tupleTypeOpt.ContainingType, BinderFlags.None);
```

Comment:
I thought I already covered that scenario in the `ObsoleteValueTuple` test. But I now realize I only have the case where VT'8 is obsolete and VT'3 is not. 
I will test the reverse too.

I was hesitant to move such logic into `TupleTypeSymbol.Create` because I think it will generate duplicate warnings when you spell out the type: `(int, int) t;`.